.github/workflows/linux.yml

```diff
@@ -131,6 +131,11 @@ jobs:
         - SWIGLANG: python
           VER: '3.13'
           CSTD: gnu99
+        - SWIGLANG: python
+          VER: '3.13'
+          PY_ABI_VER: '3.9'
+          CSTD: gnu99
+          CPPFLAGS: '-O1'
```

Comment:
`CPPFLAGS` is for C-preprocessor flags - `-O1` isn't a preprocessor flag, so should go in `CFLAGS` and/or `CXXFLAGS`.